docs/resources/ocean_right_sizing_rule.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: ocean_aws"
```

Comment:
modify the page title to ocean_right_sizing

Reply:
updated